application-task-manager-ui/src/main/resources/TaskManager/Code/TaskReportResultPage.xml

```diff
@@ -76,28 +67,28 @@
       #if ($foreach.count &gt; 1)
         #set ($spaceSql = "${spaceSql} OR ")
       #end
-      #set ($spaceSql = "$!{spaceSql}doc.space like :docspace${foreach.count} OR doc.space like CONCAT(:docspace${foreach.count}, '%')")
-      #set ($discard = $whereParams.put("docspace${foreach.count}", $space))
+      #if ($space.endsWith('.WebHome'))
+        #set ($space = $space.substring(0, $space.length() - 8))
+      #end
```

Comment:
You shouldn't rely on the reference syntax. Instead, you should resolve the reference and check its ``name`` field.